libcaja-private/caja-file-operations.c

```diff
@@ -6548,7 +6548,7 @@ mark_desktop_file_trusted (CommonJob *common,
 		new_length = length + strlen (TRUSTED_SHEBANG);
 		new_contents = g_malloc (new_length);
 
-		strcpy (new_contents, TRUSTED_SHEBANG);
+		g_strlcpy (new_contents, TRUSTED_SHEBANG, sizeof (new_contents));
```

Comment:
Use `new_length` instead of `sizeof (new_contents)`.

Reply:
done